### PR TITLE
Add test plan to docs

### LINKED ISSUE #1260: Add test description to docs
Describe required steps to test the software. This can be used as a basis for automatic tests (see #1202). Link from "Release process".

### PR DESCRIPTION
**Pull request by plata**
fixes #1260

### CODE REVIEW

File: docs/_pages/test-plan.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+permalink: /test-plan/
+toc: true
+---
+
+This page describes the test plan which is used to verify that Phoenicis works as expected.
+
+Before you start, ensure that you are in a clean environment:
+* move `.Phoenicis` to `.Phoenicis_bak`
+* checkout and pull the desired branch/revision from upstream `phoenicis`
+* do a clean build (`mvn clean package`)
```
Comment (madoar) [inline docs/_pages/test-plan.md]: I think we should also add a step: `Run all automated tests via mvn clean test`

File: docs/_pages/test-plan.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+* check that Notepad has been deleted from `.Phoenicis/containers/wineprefix`
+   
+## Library
+* open "Library" tab
+* check that "Notepad++" shortcut is not listed anymore
+* check that the shortcut files have been deleted from `.Phoenicis/shortcuts`
+
+## Settings
+* open "Settings" tab
+### User Interface
```
Comment (madoar) [inline docs/_pages/test-plan.md]: Missing line break

File: docs/_pages/test-plan.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+* open "User Interface"
+* select all themes and check that they are applied
+* change scaling
+* check that UI is scaled
+* maximize window
+* click "reset"
+* restart Phoenicis
+* check that window is not maximized and default scaling and theme are used
+### Repositories
```
Comment (madoar) [inline docs/_pages/test-plan.md]: Missing line break

File: docs/_pages/test-plan.md [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+* check that repository has been removed from list
+* click "add"
+* select "local repository"
+* use "/home/<userhome>/.Phoenicis/cache/git-XXXXXX"
+* click "finish"
+* check that new repository is listed
+* click "reset
+* restart Phoenicis
+* check that repository has been removed from list
+### File Associations
```
Comment (madoar) [inline docs/_pages/test-plan.md]: Missing line break

File: docs/_pages/test-plan.md [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+* open "File Associations"
+* check that it is empty
+### Network
```
Comment (madoar) [inline docs/_pages/test-plan.md]: Missing line break

File: docs/_pages/test-plan.md [hunk 6]
```diff
@@ -0,0 +1,184 @@
...
+* open "Network"
+* check that it is empty
+### About
```
Comment (madoar) [inline docs/_pages/test-plan.md]: Missing line break

File: docs/_pages/test-plan.md (hunk 2 above)
Comment (plata) [reply docs/_pages/test-plan.md]: Why? User Interface belongs to Settings (also for the ones below).

Comment (madoar) [reply docs/_pages/test-plan.md]: Ok, I've misread the heading level. I thought it was the same level as `Settings`.

**Pull request merged**